extension/lib/main-frame.coffee

```diff
@@ -33,8 +33,9 @@ module.exports = ->
       vimfx: createConfigAPI(vim, onShutdown)
       content
     }
-    error = config.loadFile(configDir, 'frame.js', scope)
-    return error
+    try # NOTE: per default, it tries to load file:///frame.js. remove noise from log
```

Comment:
Can you elaborate on what this does?

Reply:
config.loadFile always gives an exception, since configDir by default is `/`. this just silences the exception.

Reply:
`return null unless configDir` below `configDir = prefs.get('config_file_directory')` sounds like a better solution, then. Similar to https://github.com/akhodakivskiy/VimFx/blob/687923bd151b0fe1fad9056511cbdd509e7b6bf4/extension/lib/config.coffee#L10-L14